uber/models.py

```diff
@@ -130,7 +130,7 @@ def __init__(self, choices, **kwargs):
         TypeDecorator.__init__(self, **kwargs)
 
     def process_bind_param(self, value, dialect):
-        return value if isinstance(value, str) else ','.join(value)
+        return value if isinstance(value, str) else ','.join(list(set(value)))
```

Comment:
This change lets us add a ribbon to the ribbon column without bothering to check if the column already has it. Sets contain only unique values, so converting to a set and back removes duplicates.